### ISSUE TITLE
App crashes while taking snapshot on Nexus 4

### ISSUE THREAD

**Issue by liutingdu** [evercam/evercam-play-android#30]
To Reproduce:

Play camera O'connell Str, then take snapshot, the app always crashes and restarts

The device is Nexus 4 with Android 5.1.1

Here is the logs:

```
06-10 16:05:35.517  14809-15361/? A/libc﹕ Fatal signal 11 (SIGSEGV), code 1, fault addr 0x38 in tid 15361 (Thread-930)
06-10 16:05:35.517  14809-15361/? I/libc﹕ Suppressing debuggerd output because prctl(PR_GET_DUMPABLE)==0
06-10 16:05:35.536  15344-15344/? I/dex2oat﹕ dex2oat took 1.872s (threads: 4) arena alloc=126KB java alloc=2023KB native alloc=4MB free=546KB
06-10 16:05:35.544      564-598/? W/PackageManager﹕ Code path for pkg : com.google.android.webview changing from /data/app/com.google.android.webview-1 to /data/app/com.google.android.webview-2
06-10 16:05:35.544      564-598/? W/PackageManager﹕ Resource path for pkg : com.google.android.webview changing from /data/app/com.google.android.webview-1 to /data/app/com.google.android.webview-2
06-10 16:05:35.544      564-586/? I/ActivityManager﹕ Force stopping com.google.android.webview appid=10083 user=-1: update pkg
06-10 16:05:35.578     564-1115/? I/WindowState﹕ WIN DEATH: Window{350527e9 u0 io.evercam.androidapp/io.evercam.androidapp.CamerasActivity}
06-10 16:05:35.583      564-921/? I/WindowState﹕ WIN DEATH: Window{2da8525a u0 SurfaceView}
06-10 16:05:35.586     564-1906/? I/WindowState﹕ WIN DEATH: Window{3399a38 u0 io.evercam.androidapp/io.evercam.androidapp.video.VideoActivity}
06-10 16:05:35.619      198-198/? I/Zygote﹕ Process 14809 exited due to signal (11)
06-10 16:05:35.668      564-777/? I/ActivityManager﹕ Process io.evercam.androidapp (pid 14809) has died
06-10 16:05:35.669      564-777/? W/ActivityManager﹕ Force removing ActivityRecord{1e860720 u0 io.evercam.androidapp/.video.VideoActivity t45}: app died, no saved state
06-10 16:05:35.724      564-591/? V/WindowManager﹕ Adding window Window{cb03ef8 u0 Starting io.evercam.androidapp} at 3 of 9 (after Window{b578ebb u0 com.android.launcher/com.android.launcher2.Launcher})
06-10 16:05:35.744      564-777/? I/ActivityManager﹕ Start proc 15366:io.evercam.androidapp/u0a87 for activity io.evercam.androidapp/.CamerasActivity
06-10 16:05:35.746      564-598/? I/ActivityManager﹕ Force stopping com.google.android.webview appid=10083 user=0: pkg removed
06-10 16:05:35.774      564-564/? D/JobSchedulerService﹕ Receieved: android.intent.action.PACKAGE_REMOVED
06-10 16:05:35.774      564-564/? D/BackupManagerService﹕ Received broadcast Intent { act=android.intent.action.PACKAGE_REMOVED dat=package:com.google.android.webview flg=0x4000010 (has extras) }
06-10 16:05:35.783      564-625/? I/InputReader﹕ Reconfiguring input devices.  changes=0x00000010
06-10 16:05:35.858      564-625/? I/InputReader﹕ Reconfiguring input devices.  changes=0x00000010
06-10 16:05:35.889  15366-15366/? D/EvercamPlayApplication﹕ E-Play launched
```

**Comment by GNUDimarik** [comment]
Issue is fixed, it was really my bad, I have no idea how it could work on any device at all :-)
I tried to get data from NULL pointer.
Please merge and test fix (latest pull request)

**Comment by marcoherbst** [comment]
Early morning clear brain
Well done
M

**Comment by GNUDimarik** [comment]
You right :)
Thanks!

**Comment by liutingdu** [comment]
This is fixed in commit b6e0c802412ba87465fecff5a61b2711b084b6e2
